pkg/providers/common/common.go

```diff
@@ -26,8 +27,27 @@ const (
 	publicKeyFileName  = "eks-a-id_rsa.pub"
 )
 
-func GetAuditPolicy() string {
-	return auditPolicy
+// GetAuditPolicy returns the audit policy either v1 or v1beta1 depending on kube version.
+func GetAuditPolicy(kubeVersion v1alpha1.KubernetesVersion) (string, error) {
+	kubeVersionSemver, err := semver.New(string(kubeVersion) + ".0")
```

Comment:
Why are we appending `.0` here?

Reply:
To create a valid semver string for comparison, "1.12" does not have a patch value for valid semver string.

Reply:
add a comment describing this